app/main.py

```diff
@@ -16,4 +16,18 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [], "immutable": []}
+variables = [lucky_number,
+             pi,
+             one_is_a_prime_number,
+             name,
+             my_favourite_films,
+             profile_info,
+             marks,
+             collection_of_coins]
+
+for value in variables:
+    if isinstance(value, (list, dict, set)):
+        sorted_variables["mutable"].append(value)
+    else:
+        sorted_variables["immutable"].append(value)
```

Comment:
No additional logic is required - just pass the variables to the correct list in the dictionary.